File: DeanerySystem/DeanerySystem.UI/Models/Education/Schedule/LessonGroupModel.cs
using System.Collections.Generic;

namespace DeanerySystem.UI.Models.Education.Schedule {
	public class LessonGroupModel {
		public LessonGroupModel() {
			Lessons = new List<LessonModel>();
		}

		public int GroupId { get; set; }
		public List<LessonModel> Lessons { get; set; }
	}
}

File: DeanerySystem/DeanerySystem.ClientSite.Schedule/Models/LessonGroupModel.ts
module DeanerySystem.ClientSide.Schedule.Models {
	export class LessonGroupModel {
		public GroupId: number;
		public FirstRowLesson: LessonModel;
		public SecondRowLesson: LessonModel;
		public IsSolid: boolean;

		constructor() { }
	}
}
